firefox-ios/Storage/Rust/RustFirefoxSuggestion.swift

```diff
@@ -64,6 +64,7 @@ public struct RustFirefoxSuggestion: Equatable {
             impressionUrlString,
             clickUrlString,
             _,
+            _,
```

Comment:
looks correct, but I wonder if there's a way to re-write this to make it less fragile.  In Rust, I'd do something like `if let Amp::Suggestion { title, urlString, iconBytes, ... } = suggestion { ... }`, which should work even if fields are added or re-ordered.  Is there anything like that in Swift?